ui/pages/settings/developer-options-tab/developer-options-tab.tsx

```diff
@@ -91,6 +96,13 @@ const DeveloperOptionsTab = () => {
     setIsServiceWorkerKeptAlive(value);
   };
 
+  const handleToggleEnableConfirmationsRedesign = async (
```

Comment:
perhaps `setConfirmationsRedesignEnabled` as we're setting instead of toggling?

Reply:
we toggle on the UI, happy to rename if necessary

Reply:
This is totally optional but the reason I think `set` is a better fit here is that `toggle` functions are usually associated with flipping the value independently of what it is at the moment.

for example

```js
const toggleEnableConfirmationsRedesign = async (): Promise<void> => {
    const previousValue = useSelector(isRedesignedConfirmationsFeatureEnabled);

    await dispatch(setRedesignedConfirmationsDeveloperEnabled(!previousValue));
    await setIsRedesignedConfirmationsFeatureEnabled(!previousValue);
}
```

However, in this case the function is receiving an explicit value as an argument, so it seems to be a `set` function.

In other words, there is indeed a feature toggle, but the function is not toggling, it's setting.

Reply:
done